packages/flutter/lib/src/material/material.dart

```diff
@@ -617,6 +630,9 @@ abstract class InkFeature {
 
   bool _debugDisposed = false;
 
+  /// Whether or not visual reaction is activated.
+  bool visible = true;
```

Comment:
The docs for this should be clearer about the responsibilities of the caller when this is changed. (For example, do you have to setState on something?)

Reply:
Updated.